contracts/allowListExamples/AllowListOffChainManaged.sol

```diff
@@ -1,11 +1,12 @@
 pragma solidity >=0.6.8;
 import "@openzeppelin/contracts/access/Ownable.sol";
 import "../interfaces/AllowListVerifier.sol";
+import "@openzeppelin/contracts/introspection/ERC165.sol";
 
 // Idea was first mentioned in the blog:
 // https://medium.com/@PhABC/off-chain-whitelist-with-on-chain-verification-for-ethereum-smart-contracts-1563ca4b8f11
 
-contract AllowListOffChainManaged is Ownable {
+contract AllowListOffChainManaged is Ownable, ERC165 {
```

Comment:
@fedgiac 
I am thinking about using ERC165, in order to validate quickly whether we have a correct contract deployed for the allowManager.
This is very handy in the initiate_auction script.
Though, it feels like there should be an easier solution to my problem... :( 

Are you aware of a possible check, whether a contract implements the right interface without modifying the contract - like without inheriting ERRC165? Theoretically, ethers could check that all the functions are callable... somehow.